package.json

```diff
@@ -293,44 +298,34 @@
                     "group": "2_appServiceControlCommands@4"
                 },
                 {
-                    "command": "appService.DeployZipPackage",
```

Comment:
Why doesn't the App Service node have the deploy command?

Reply:
Per discussion with @fiveisprime we thought this was a confusing entry point and replaced it with the one on the title bar. The node entry point implied you were deploying the app service itself (as opposed to deploying TO the app service)

Reply:
What does it mean by "deploying the app service itself"? The app service is already there, why would someone think the service itself needs to be deployed again?

Reply:
I'll approve for now. We can discuss this a bit more offline.